_guide_topics/rhinocommon/compute/compute-getting-started.md

```diff
@@ -0,0 +1,306 @@
+---
+title: Rhino Compute Service (Windows)
+description: This guide covers all the necessary tools required to get started with the RHino Compute Service
+authors: ['Steve Baer', 'Scott Davidson']
+author_contacts: ['stevebaer', 'scottd']
+sdk: ['RhinoCommon', 'Compute']
+languages: ['C#']
+platforms: ['Windows']
+categories: ['Getting Started']
+origin:
+order: 1
+keywords: ['first', 'RhinoCommon', 'Plugin', 'compute']
+layout: toc-guide-page
+---
+
+
+By the end of this guide, you should have all the tools installed necessary for using the [Rhino Compute Service](https://www.rhino3d.com/compute).
+
+## Prerequisites
+
+This guide presumes you have an:
+
+- A PC running Microsoft Windows 7 or later.
+- [Rhino for Windows](http://www.rhino3d.com/download)
+- One of the IDEs from [Microsoft Visual Studio](https://visualstudio.microsoft.com/)
+  - [Visual Studio Code 2017](https://code.visualstudio.com/?wt.mc_id=DX_841432) - Free, Open Source.
+  - Or, [Visual Studio 2017](https://visualstudio.microsoft.com/vs/whatsnew/)
+- Internet Access
+
+**note:** It is recommended that you install the *Typical* installation.
+
+---
+
+## Setting up a Compute Project in Visual Studio
+
+There are a few tools which are essential to communicate with the Compute server. These include:
```

Comment:
I don't think it's necessary to list these "prerequisites" at this point. It's OK to bring up the NuGet packages and RhinoCompute.cs during the course of the guide.